apps/projects/app/components/Panel/SubmitWork/SubmitWork.js

```diff
@@ -1,19 +1,147 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import styled from 'styled-components'
 
-const SubmitWork = ({ issues, rate, onSubmit }) => <div>Submit Work</div>
-
-SubmitWork.propTypes = {
-  /** array of issues to allocate bounties on */
-  issues: PropTypes.arrayOf(
-    PropTypes.shape({
-      id,
-      level,
-    })
-  ),
-  /** base rate in pennies */
-  rate: PropTypes.number,
-  onSubmit: PropTypes.func,
+import {
+  Field,
+  Text,
+  TextInput,
+  theme,
+  Info,
+  SafeLink,
+  IconAttention
+} from '@aragon/ui'
+
+import { Form, FormField, FieldTitle } from '../../Form'
+import { IconGitHub, CheckButton } from '../../Shared'
+
+class SubmitWork extends React.Component {
+
+  static propTypes = {
+    issue: PropTypes.object.isRequired
+  }
+
+  state = {
+    proof: '',
+    comments: '',
+    hours: 0,
+    ack1: false,
+    ack2: false
+  }
+
+  changeField = ({ target: { name, value } }) => this.setState({ [name]: value })
+  setAck1 = () => this.setState(prevState => ({ ack1: !prevState.ack1 }))
+  setAck2 = () => this.setState(prevState => ({ ack2: !prevState.ack2 }))
+
+  onSubmitWork = () => {
+    console.log('Submit', this.state)
+  }
+
+  canSubmit = () => !(this.state.ack1 && this.state.ack2 && this.state.proof)
+
+  render() {
+    // TODO: replace with props
+    const account = 'accplaceholder'
+
+    const { title, repo, number, url } = this.props.issue
+
+    return (
+      <Form
+        onSubmit={this.onSubmitWork}
+        submitText="Submit Work"
+        noSeparator
+        submitDisabled={this.canSubmit()}
+      >
+        <Text>{title}</Text>
+        
+        <SafeLink
+          href={url}
+          target="_blank"
+          style={{ textDecoration: 'none', color: '#21AAE7' }}
+        >
+          <IssueLinkRow>
+            <IconGitHub color="#21AAE7" width='14px' height='14px' />
+            <Text style={{ marginLeft: '6px'}}>{repo} #{number}</Text>
+          </IssueLinkRow>
+        </SafeLink>
+
+        <FormField
+          label="Proof of Work"
+          required
+          input={
+            <TextInput.Multiline
+              name='proof'
+              rows={3}
+              style={{ resize: 'none', height: 'auto' }}
+              onChange={this.changeField}
+              placeholder="Please link the Github Pull Request or an alternative proof of work if requested."
+              wide
+            />
+          }
+        />
+        <FormField
+          label="Additional Comments"
+          input={
+            <TextInput.Multiline
+              name='comments'
+              rows={5}
+              style={{ resize: 'none', height: 'auto' }}
+              onChange={this.changeField}
+              placeholder="Do you have any other comments or details you would like to provide that hasn’t already been described elsewhere?"
```

Comment:
I would shorten this placeholder, doesn't need to be this long/polite.

Reply:
Shortened.